velox/type/HugeInt.h

```diff
@@ -56,6 +56,13 @@ class HugeInt {
   }
 
   static int128_t parse(const std::string& str);
+
+  /// Because value exceeds maxium int64_t cannot write, add this helper
+  /// function to test. It can represent -9223372036854775817223372036854775808
+  /// to 9223372036854775816223372036854775807.
+  /// Example Usage:
+  /// combine(1, 13579) return 1'000'000'000'000'013'579
+  static int128_t combine(int64_t hi, int64_t lo);
```

Comment:
It would be nice to explain how this method differs from existing 'build' method? Is it producing a value that's equal to hi * 10^18 + lo?  hi and lo here would not match the result of calling upper() and lower() on the result. Would it be confusing? Are there gaps in the range of values this can produce?

Reply:
I rename the argument to make it more clear.
Yes, it cannot represents the max int128, the max value is all the input argument is INT64_MAX, the range is clarified in comments.